integration_tests/src/main/python/window_function_test.py

```diff
@@ -241,6 +241,7 @@ def _gen_data_for_collect(nullable=True):
             ['child_decimal', DecimalGen(precision=8, scale=3)]]))]
 
 
+
```

Comment:
Is this change intentional?

Reply:
The test was originally in a different spot as I worked on it based off of the out of core sort patch.  Then when I upmerged and moved it I think I left an extra blank line.